package-glue-job.sh

```diff
@@ -47,45 +47,37 @@ then
     pipenv requirements > requirements.txt
 
     echo "Installing dependencies..."
-    pip install -q -r requirements.txt --no-deps -t .
+    python -m pip install -q -r requirements.txt --no-deps -t .
 else
     echo "No Pipfile found, assuming no external dependencies are being used."
 fi
 
 # Create Wheel Package
 
-echo "* Creting setup.py file"
-cat >setup.py <<EOL
-from setuptools import setup, find_namespace_packages
-import glob, os
-
-# The the current filename without the extension.
-def get_current_py_module():
-    current_file_name = os.path.basename(__file__)
-    return os.path.splitext(current_file_name)[0]
+echo "* Creating Wheel package"
 
+read -r -d '' CMD <<EOF
+from setuptools import setup, find_namespace_packages
+from glob import glob
+from os.path import basename, splitext
 
-# Returns the filename of other .py files in the current directory without the extensions.
+# Returns the filename of all .py files in the current directory without their extensions.
 def find_py_modules():
-    cwd = os.getcwd()
-    file_paths = glob.glob(cwd + "/*.py")
-    file_names = [os.path.basename(file_path) for file_path in file_paths]
-    py_modules = [os.path.splitext(file_name)[0] for file_name in file_names]
-    py_modules.remove(get_current_py_module())
+    file_paths = glob("*.py")
+    file_names = [basename(file_path) for file_path in file_paths]
+    py_modules = [splitext(file_name)[0] for file_name in file_names]
     return py_modules
 
 setup(
     name="deps",
     version="1.0",
     packages=find_namespace_packages(),
     py_modules=find_py_modules(),
+    setup_requires=['wheel'],
```

Comment:
This auto-installs wheel if missing (which was the case for me). There may be some caveats (see https://stackoverflow.com/a/54833684), but .. it worked well for me.